astrobee/config/management/executive.config

```diff
@@ -24,7 +24,7 @@ led_service_available_timeout = 10
 
 gs_command_timeout = 4
 
-motion_feedback_timeout = 1
+motion_feedback_timeout = 5
```

Comment:
replanning takes longer than 1 second sometimes because of the iterations

Reply:
what do you mean by the iterations?

Reply:
replanning has a 1s delay for the octomap to be updated and forget the non-visible obstacles